src/CoreBundle/Resources/public/javascript/tableEditor.js

```diff
@@ -937,6 +937,30 @@ CHAMELEON.CORE.MTTableEditor.idButtonCopyToClipboard = function () {
     });
 };
 
+CHAMELEON.CORE.MTTableEditor.initSwitchToEntryButtons = function () {
+    document.querySelectorAll("[data-link-for-select]").forEach(function(linkButton) {
+        const selectElement = document.querySelector('select[name="' + linkButton.dataset.linkForSelect + '"]');
+        if (null === selectElement) {
+            return;
+        }
+
+        // NOTE a change listener on the selectElement (= "the other way around") does not work as that is masked by select2 pseudo element (TODO remove select2)
+
+        linkButton.addEventListener("click", function(e) {
+            if ("" === selectElement.value) {
+                e.preventDefault();
+
+                return;
+            }
+
+            const href = new URL(linkButton.getAttribute("href"), document.location.origin);
+            href.searchParams.set('id', selectElement.value);
+
+            linkButton.setAttribute("href", href.toString());
+        });
+    });
+};
```

Comment:
Why are we modifying the links after the fact. Can't this logic live directly in the twig template that introduced those buttons?

Reply:
If the select option changes so must the target of the link (button).